### DISCUSSION TITLE
"Bug" com as perícias

### DISCUSSION THREAD

**Discussion by MeetLime291621** [alyssapiresfernandescefet/openrpg#7] [Show and tell]
Estava alterando minha perícias para o sistema que vou usar em minha campanha e vi que havia feito um erro, tentei corrigir, mas não conseguia editar mais nenhuma perícia. Mesmo excluindo todas as perícias eu não consegui nem criá-las de novo.
![imagem_2022-06-21_183827101](https://user-images.githubusercontent.com/107946712/174901594-eb578783-069e-4c8f-a921-3a75dbe56e01.png)
Descobri que para resolver isso era necessário que haja pelo menos uma "Especialidade" (mesmo o seu RPG não tendo esta mecânica). Não sei se é um bug ou é proposital, mas caso alguém tenha esse empecilho esta é a solução.
![image](https://user-images.githubusercontent.com/107946712/174902061-e064b593-3201-4391-a542-7955a707e5db.png)
Agora consigo voltar a mudanças das perícias.
Obrigado pela atenção.

**Comment by lucaspiresfernandes** [comment]
Olá! @MeetLime291621! Sim, isso é um erro e não deveria acontecer, visto que perícias deveriam existir independente de especializações. Já estou trabalhando para consertar esse erro.

**Reply by lucaspiresfernandes** [reply]
[Atualização]

Bug resolvido no commit 40d09572399b09459a391147eaf9ce8523dfff73 ! Atualize seu Open RPG.

**Reply by MeetLime291621** [reply]
Muito Obrigado!

**Comment by MeetLime291621** [comment]
Tenho só uma dúvida, como faço para atualizar? Pesquisei no Google e apareceu algo sobre "Pull Requests", mas não sei exatamente o que isso significa.
Minha intuição me diz que é nessa opção (Fetch and Merge) que eu atualizo o meu fork, mas tenho medo de corromper algo.
![image](https://user-images.githubusercontent.com/107946712/175074588-dd7d59c2-0479-4f02-b444-146536f8721e.png)

**Reply by lucaspiresfernandes** [reply]
Olá! Pra atualizar seu site do Open RPG, você primeiro precisa atualizar seu repositório e depois fazer o redeploy na Heroku.

Então, pra atualizar o seu repositório você clica nesse botão mesmo, "Fetch and merge"! Esse botão irá baixar todos os arquivos novos e modificados e vai aplicar ao seu repositório.

Depois, você vai na Heroku e faz o redeploy!

Se tiver mais dúvidas, acesse esse tutorial de como atualizar: https://github.com/alyssapiresfernandescefet/openrpg/blob/main/docs/update.md

**Reply by MeetLime291621** [reply]
Segui o procedimento e deu tudo certo, obrigado!
